app/forms/lead_provider_form.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class LeadProviderForm
+  include ActiveModel::Model
+
+  attr_accessor :name, :cip, :cohorts
+
+  def available_cips
+    Cip.all
+  end
+
+  def available_cohorts
+    Cohort.all
+  end
+
+  def chosen_cohorts
+    Cohort.where(id: cohorts)
+  end
+
+  def chosen_cohort_names
+    cohorts.map { |id| Cohort.find(id).display_name }
+           .join(", ")
+  end
+
+  def chosen_cip
+    Cip.find(cip)
+  end
+
+  def chosen_cip_name
+    Cip.find(cip).name
+  end
```

Comment:
At the risk of being a pain in the ass, maybe this method isn't doing enough to justify its existence.

Reply:
Having `Cip.find(@lead_provider_form.cip).name` in the view just seemed ugly to me 🤷

Reply:
How about `@lead_provider_form.chosen_cip.name`? (the method above this one)